tests/cc/CMakeLists.txt

```diff
@@ -1,5 +1,19 @@
 # Copyright (c) PLUMgrid, Inc.
 # Licensed under the Apache License, Version 2.0 (the "License")
+
+find_program(ARPING arping)
+if(ARPING STREQUAL "ARPING-NOTFOUND")
+  message(WARNING "Required test program 'arping' not found")
```

Comment:
Yes. arping/iperf/netperf are used in python codes in several tests. They
will be needed in test running time.

On Wed, Aug 5, 2015 at 3:14 PM, Brenden Blanco <notifications@github.com>
wrote:

> In tests/cc/CMakeLists.txt
> <https://github.com/iovisor/bcc/pull/118#discussion_r36361192>:
>
> > @@ -1,5 +1,19 @@
> >  # Copyright (c) PLUMgrid, Inc.
> >  # Licensed under the Apache License, Version 2.0 (the "License")
> > +
> > +find_program(ARPING arping)
> > +if(ARPING STREQUAL "ARPING-NOTFOUND")
> > +  message(WARNING "Required test program 'arping' not found")
>
> I guess...it's "required" for test but not for build?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/iovisor/bcc/pull/118/files#r36361192>.
>